content/zh/docs/installing-on-linux/high-availability-configurations/ha-configuration.md

```diff
@@ -89,15 +89,15 @@ curl -sfL https://get-kk.kubesphere.io | VERSION=v1.0.1 sh -
 chmod +x kk
 ```
 
-创建包含默认配置的示例配置文件。这里使用 Kubernetes v1.17.9 作为示例。
+创建包含默认配置的示例配置文件。这里使用 Kubernetes v1.20.4 作为示例。
 
 ```bash
-./kk create config --with-kubesphere v3.0.0 --with-kubernetes v1.17.9
+./kk create config --with-kubesphere v3.1.0 --with-kubernetes v1.20.4
 ```
 
 {{< notice note >}}
 
-- 在 KubeSphere 上充分测试过的 Kubernetes 版本：v1.15.12、v1.16.13、v1.17.9（默认）以及 v1.18.6。
+- 安装 KubeSphere v3.1.0 的建议 Kubernetes 版本：v1.17.9，v1.18.8，v1.19.8 以及 v1.20.4。如果不指定 Kubernetes 版本，KubeKey 将默认安装 Kubernetes v1.19.8。有关受支持的 Kubernetes 版本的更多信息，请参见[支持矩阵](../../docs/installing-on-linux/introduction/kubekey/#支持矩阵)。
```

Comment:
Change the relative path. You use `docs` already. Delete `../..` at the beginning. Change this across this pr.